front/src/styles/common.css

```diff
@@ -0,0 +1,33 @@
+.second-color {
+    color: #FFA21C;
+}
+
+.main-color,
+button.main-color,
+a.main-color.el-link--default {
+    color: white;
+}
+
+.color-background,
+button.color-background,
+ul.color-background {
+    background-color: #6663FF;
+}
+
+button.second-color {
+    color: #6663FF;
+}
+
+.footer-color-background,
+button.footer-color-background {
```

Comment:
did you read about css variables? Please, add the color values as variables into variables.css and use these variables instead of static text values in these classes